app/components/course-admin/submissions-page/submission-details/header-container.hbs

```diff
@@ -103,6 +103,19 @@
           </div>
         </CourseAdmin::SubmissionsPage::SubmissionDetails::HeaderContainerRow>
 
+        <CourseAdmin::SubmissionsPage::SubmissionDetails::HeaderContainerRow @title="Tree SHA" data-test-tree-sha>
+          <div class="flex items-center">
+            <span class="mr-2 text-gray-600">
+              {{this.shortSubmissionTreeSha}}
```

Comment:
```suggestion
              {{or this.shortSubmissionTreeSha "Unknown"}}
```

(This way you don't have the weird naming where "shortSubmissionTreeSha" isn't actually a tree SHA)